app/build.gradle

```diff
@@ -54,15 +54,23 @@ repositories {
 }
 
 dependencies {
+    def room_version = "1.1.1"
```

Comment:
What does this variable represent?
Also, add an empty line to separate kind groups